### DISCUSSION TITLE
热度下去了

### DISCUSSION THREAD

**Discussion by yaojun170** [WeNeedHome/SummaryOfLoanSuspension#465] [General]
这样下去不到一周就没声音了

**Comment by sqpk** [comment]
不是热度下去了，是国内封杀的太厉害，以至于现在断贷的不同楼盘业主没办法联合起来，绝大部分人也并不知情，希望停贷的也只能够坚持，同心协力，虽然我的没烂尾，但仍希望他能捍卫自己的权益

**Reply by xiejun946** [reply]
我的烂尾了，说停工了也没停工，交一个不通水电气的给我们，啥商业配套都没有，是好很多，但好像也没什么可以做的。明确半年一年办不了证

**Comment by yalo-ddaa** [comment]
没有吧，各大平台都还能看到很多讨论。不知道为什么仓库昨晚被repo主限制发言和提交PR了。

**Reply by sqpk** [reply]
怕半夜整出来什么幺蛾子出来，

**Reply by DarkForestBlack** [reply]
有个啥雀在discussion里说大当家的坏话，估计repo主怕喝茶给禁言了吧

**Reply by sqpk** [reply]
> 

咳咳，确实不咋地，不然为啥郑州710储户喊李不喊他呢，不过谨言慎行是好的，希望下半年有惊喜

**Reply by DarkForestBlack** [reply]
> 

感恩就完了

**Reply by Edan233** [reply]
知乎搜不到了

**Reply by tencentbook** [reply]
国内的讨论链接好几个都被封了啊

**Comment by C-Joey** [comment]
真想搞维权 就不要用什么微信及任何国内的社交软件 热度下降是因为对相关链接的封杀以及社交软件自动过滤敏感词 国家机器已经开动起来了

**Comment by dongyc519** [comment]
hn的闹事，没见过guonei媒体有一篇报过，这也是悲哀。

**Reply by C-Joey** [reply]
“hn”、“guonei”是什么意思 是自我阉割吗 建议少用国内社交软件 容易导致自我阉割

**Reply by jingzz1** [reply]
河南，国内，在这里不用那么隐晦，不过我想你应该习惯了。

**Reply by Jesse3692** [reply]
哎  不知不觉的

**Reply by tencentbook** [reply]
河南的维权行动，没见过国内媒体有一篇报过，这也是悲哀。

**Reply by yalo-ddaa** [reply]
现在有了，有一家媒体在新闻视频的最后一秒钟掺了一句”储户与不明人士发生冲突“，很短但评论都是发”重点在最后一秒“的。

**Comment by liuwenwu520** [comment]
只有业主们到公安局申请游行了，这么热的天也是难啊

**Reply by Jesse3692** [reply]
能给批吗

**Reply by selphy65** [reply]
你疯了吧。以我对共产党的了解，不开坦克就不错了。你还送人头。公安局是不会批准的。信我的，一定不要暴露自己的身份。大家齐心协力

**Comment by uueer** [comment]
哈哈哈

**Comment by yangtuqiang** [comment]
有媒体敢出声么，他们出声不就只会说停贷犯法

**Comment by trunklater** [comment]
国内大大小小的浏览器都打不开这个地址。各个银行密集发布公告声明“数目不大，风险较小。”
这不明摆着没啥结果的嘛。
